<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="atlas" >
  <xacro:include filename="../components/atlas_v5_minimal_contact.urdf" />
  <xacro:include filename="../../common_components/multisense_sl/multisense_sl_no_collision.urdf" />
  <xacro:include filename="./atlas_v5_common.xacro" />

  <xacro:robotiq_camera  side_letter="l" reflect="1" parent="l_hand_force_torque" >
    <origin xyz="0.0  0.0 0.0" rpy="0.0 0  0.0"/>
  </xacro:robotiq_camera>
  <xacro:robotiq_camera  side_letter="r" reflect="-1" parent="r_hand_force_torque" >
    <origin xyz="0.0  0.0 0.0" rpy="0.0 0  0.0"/>
  </xacro:robotiq_camera>
</robot>
